pyproject.toml

```diff
@@ -22,6 +22,7 @@ dependencies = [
   "numpy>=1.16",
   "scipy>=1.3",
   "pints>=0.5",
+  "bpx>=0.4",
```

Comment:
We might be able to relax this dependancy, did you happen to test with earlier versions of BPX?

Reply:
I didn't test with earlier versions but since this functionality is taken from PyBaMM, I think it's best to have the same restriction here. PyBaMM has: `"bpx>=0.4.0"`.